pkg/cmd/roachtest/tests/typeorm.go

```diff
@@ -25,7 +25,10 @@ import (
 )
 
 var typeORMReleaseTagRegex = regexp.MustCompile(`^(?P<major>\d+)\.(?P<minor>\d+)\.(?P<point>\d+)$`)
-var supportedTypeORMRelease = "0.3.17"
+
+// Use 0.3.18 from the upstream repo once it is released.
```

Comment:
Should we file an issue for this and leave a TODO comment here?

Reply:
IMO there's not actually any urgency in updating to the stable release. Whenever we get around to updating the ORMs we test, we can handle this one too. (I feel like it's similar to why we don't make tracking issues for updating each ORM under test until we see a need to)